databricks/koalas/frame.py

```diff
@@ -8331,6 +8331,104 @@ def quantile(self, q=0.5, axis=0, numeric_only=True, accuracy=10000):
 
         return DataFrame(internal) if not result_as_series else DataFrame(internal).T[key]
 
+    def query(self, expr, inplace=False):
+        """
+        Query the columns of a DataFrame with a boolean expression.
+
+        .. note::
+            * Using variables in the environment with `@` is not supported.
+            * Internal columns that starting with __. are able to access,
+              however, they are not supposed to be accessed.
+            * This delegates to Spark SQL so the syntax follows Spark SQL
+            * If you want the exactly same syntax with pandas' you can work around
+              by using :meth:`DataFrame.map_in_pandas`. See the example below.
+
+                >>> df = ks.DataFrame([(1, 2), (3, 4), (5, 6)], columns=['A', 'B'])
+                >>> df
+                   A  B
+                0  1  2
+                1  3  4
+                2  5  6
+                >>> num = 1
+                >>> df.map_in_pandas(lambda pdf: pdf.query('A > @num'))  # doctest: +SKIP
```

Comment:
@itholic can you enable this since `map_in_pandas` is merged?

Reply:
@HyukjinKwon sure, i'm waiting for!